package.json

```diff
@@ -62,7 +63,7 @@
     "eslint-plugin-react": "^7.17.0",
     "eslint-plugin-react-hooks": "^1.7.0",
     "react-native-svg-transformer": "^0.14.3",
-    "typescript": "^3.6.3"
+    "typescript": "^3.8.3"
```

Comment:
was there a reason to bump the typescript version?

Reply:
Oh when I was I was watching the CI / build (pull_request) checks run I got a notification that my typescript was out of date so I ran npm i typescript to update it. Should I revert it back to "^3.6.3"?

Reply:
ya i think you should revert. though it's doubtful updating typescript will break anything, it's good practice to separate these concerns.

we can make note of it though for a future PR.

Reply:
Reverted the file to before the typescript update.